product_docs/docs/pgd/5/rel_notes/pgd_5.1.0_rel_notes.mdx

```diff
@@ -6,6 +6,29 @@ navTitle: "Version 5.1.0"
 EDB Postgres Distributed version 5.1.0 is a patch version of EDB Postgres Distributed.
 This version addresses security vulnerabilities in dependencies for PGD-Proxy and PGD-CLI.
 
-| Component | Version | Type    | Description                                          |
-|-----------|---------|---------|------------------------------------------------------|
-| PGD       | 5.1.0   | Feature | Add pid to the log message emitted upon writer process death (BDR-2534) |
+| Component | Version | Type            | Description                                          |
+|-----------|---------|-----------------|------------------------------------------------------|
+| PGD       | 5.1.0   | Feature         | Add pid to the log message emitted upon writer process death (BDR-2534) |
+| PGD       | 5.1.0   | Bug fix         | Fix join and replication when SDW and standby_slot_names are set (BDR-3176, RT89702, RT89536) |
+| PGD       | 5.1.0   | Bug fix         | Fix spurious warnings when processing sequence options (BDR-3295, RT90668) |
+| PGD       | 5.1.0   | Bug fix         | Fix upgrades for nodes with CRDTs (BDR-3302) |
+| PGD       | 5.1.0   | Feature         | Add group name in the bdr.event_summary view (BDR-3315) |
+| PGD       | 5.1.0   | Bug fix         | Adjust lag tracking parameters for LCRs from pg_stat_replication (BDR-3040) |
+| PGD       | 5.1.0   | Bug fix         | Adjust node routing options defaults based on node kind (BDR-3029)  for example witness nodes are not marked as candidates for receiving writes |
+| PGD       | 5.1.0   | Feature         | Add SLES support (BDR-3101)   SLES 12sp5 and SLES 15sp4 are now supported |
+| PGD       | 5.1.0   | Feature         | Convert all sequences to "distributed" during create node (BDR-2890)  |
+| PGD       | 5.1.0   | Feature         | Add subscription status column to the group subscription summary view. (BDR-2198)  This will help distinguish whether NULL values is due to a node being down or subscription being disabled.|
```

Comment:
NULL values are due to...

or a subscription being disabled.

(Github's not letting me suggest again, sorry)